src/faultlog/unresolved_pels.hpp

```diff
@@ -26,19 +26,23 @@ class UnresolvedPELs
   public:
     /** @brief Get count of unresolved pels with deconfig bit set
      *  @param[in] bus - D-Bus to attach to
+     *  @param[in] hostPowerOn - flag to specify if IPL is in progress
      *
      *  @return 0 if no records are found else count of records
      */
-    static int getCount(sdbusplus::bus::bus& bus);
+    static int getCount(sdbusplus::bus::bus& bus, bool hostPowerOn);
```

Comment:
`const bool hostPowerOn` if we are not modifying it at all

Reply:
who cares it is pass by variable